src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -44,7 +44,7 @@ import Sentry
 
 SentrySDK.start { options in
     options.dsn = "___PUBLIC_DSN___"
-    options.enableAutoUIPerformanceTracking = false
+    options.enableAutoPerformanceTracking = false
```

Comment:
When this goes GA, could we please also add this option to the Basic Options file, noting that it's an opt-out and that a sample rate ("adding performance") must be done so this takes effect? File is options.mdx. And we update it using these [guidelines](https://docs.sentry.io/contributing/approach/sdk-docs/write-configuration/#basic-options).